source/javascripts/components/_Step.js

```diff
@@ -36,9 +36,27 @@ import { normalizeIconUrl } from "./StepItem/StepItem";
 				return this.id;
 			}
 
-			return this.cvs;
+			return diplayNameFromCvs(this.displayCvs());
 		};
 
+		Step.prototype.displayCvs = function() {
+			return this.cvs.replace(/^(git|path)::/g, "");
+		}
+
+		Step.prototype.displayTooltip = function() {
+			return `${this.displayName()}<br>${this.displayCvs()}`;
```

Comment:
ahh sorry just realised, we do not have ES6 support in plain js files, can you please backport it to ES5? :)

Reply:
Ohh, don't we have a Babel loader for taking care of the transpile for js files?

Reply:
nope and that's intentional to force people to use typescript instead and speed up the migration 😄

Reply:
OK, got it :)